thesis-de/content-de.tex

```diff
@@ -1685,6 +1685,57 @@ \subsection{Durchführung der Evaluierung}
 \end{center}
 \nl%
 
+\subsection{Zusammenspiel API und Frontend}
+\label{subsec:zusammenspiel_api_frontend}
+Nachdem die Evaluierung der Datenvisualisierung im
+Abschnitt~\ref{subsec:durchfuehrung_evaluierung_datenvisualisierung}
+abgeschlossen war, wurde dessen Zusammenspiel mit der API überprüft.
+Dabei schaute sich Herr Luis speziell die Syntax zwischen den beiden
+Softwareprodukten an. Die Syntax definiert dabei den Aufbau einer Datenanfrage
```

Comment:
Ich kann nicht in dem Abschnitt Schnittstelle - API, die beiden Wörter Schnitstelle oder API verwenden... 
Aber ja ich habe es angepasst